.github/labeler.yml

```diff
@@ -0,0 +1,2 @@
+v1.0:
+- base-branch: 'release-1.0'
```

Comment:
Isn't this label called `v1.0` in `labels.yaml`?

Reply:
That's what it is on line 1, yeah. The branch is named `release-1.0` though.

Reply:
oh OK I read the config file the other way around